package.json

```diff
@@ -94,7 +94,7 @@
     "moment": "^2.24.0",
     "node-sass": "^4.12.0",
     "npm-run-all": "^4.1.5",
-    "postcss-cli-simple": "^1.0.3",
+    "postcss-cli-simple": "^3.0.0",
```

Comment:
https://github.com/pirxpilot/postcss-cli/releases/tag/3.0.0

Reply:
Hm, given this is getting less than 200 installs a week (https://www.npmjs.com/package/postcss-cli-simple) should we be using the official https://www.npmjs.com/package/postcss-cli instead?

Reply:
Sure. I thought about that as well but I couldn't decide if we needed the full package (vs the "simple" version). Will update the PR to replace `postcass-cli-simple` with `postcass-cli`.